iosxr_iso2vbox.py

```diff
@@ -124,10 +149,10 @@ def main(argv):
     # Handle Input ISO (Local or URI)
     if re.search(':/', args.ISO_FILE):
         # URI Image
-        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
+        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
```

Comment:
I need a little more context or an example? What I've done is common and flake8 is happy.